src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -32,6 +32,7 @@ case class Settings(ownerId: String,
                     otherAccessDoi: Boolean = false,
                     stageFileDataAsRedirectDatastreams: Boolean = false,
                     fileDataRedirectBaseUrl: Option[URL] = None,
+                    stubAVfiles: Boolean = false,
                     disciplines: Map[String, String]) {
```

Comment:
Notice that changes in this class will affect easy-ingest-flow. The API changes, so you must change it there as well.